CONTRIBUTING.md

```diff
@@ -21,14 +21,11 @@ Date:   Thu Feb 2 11:41:15 2018 -0800
 
     Signed-off-by: Joe Smith <joe.smith@example.com>
 
-## How to Contribute to an Existing Chart
+## How to debug deployments
 
-1. Fork this repository, develop and test your Chart changes.
-2. Remember to sign off your commits as described in the "Sign Your Work" chapter.
-3. Ensure your Chart changes follow the technical and documentation guidelines, described below.
-4. Submit a pull request.
-
-NOTE: In order to make testing and merging of PRs easier, please submit changes to multiple charts in separate PRs.
+```
+helmfile template --no-color -f clusters/<cluster-name>.yaml -l name=<release-name>
+```
```

Comment:
As I proposed in https://github.com/jenkins-infra/kubernetes-management/pull/3339#pullrequestreview-1218779000, could you, please:

> - Keep the section "How to Contribute to an Existing Chart" but change it with:
>   - Rename to "How to Contribute"
>   - Add a first bullet telling to start by opening an issue on github.com/jenkins-infra/helpdesk explaining the goal, before the PR
>   - Add another bullet telling that the jenkins-infra chart contributions are described in another repository (link: https://github.com/jenkins-infra/helm-charts/blob/main/CONTRIBUTING.md)

Reply:
> As I proposed in [#3339 (review)](https://github.com/jenkins-infra/kubernetes-management/pull/3339#pullrequestreview-1218779000), could you, please:
> 
> > * Keep the section "How to Contribute to an Existing Chart" but change it with:
> >   
> >   * Rename to "How to Contribute"
> >   * Add a first bullet telling to start by opening an issue on github.com/jenkins-infra/helpdesk explaining the goal, before the PR
> >   * Add another bullet telling that the jenkins-infra chart contributions are described in another repository (link: https://github.com/jenkins-infra/helm-charts/blob/main/CONTRIBUTING.md)

yes I working on this, apologize for delay

Reply:
@ADITYADAS1999 no worries, there is no emergency in this matter. Take the time you need, don't stress for this. My message was only to ensure that information was not lost since we work asynchrounously.

Reply:
> @ADITYADAS1999 no worries, there is no emergency in this matter. Take the time you need, don't stress for this. My message was only to ensure that information was not lost since we work asynchrounously.

thanks [dduportal](https://github.com/dduportal) 😃, you very super active in community.